services/mlmodel/tflite_cpu/tflite_cpu.go

```diff
@@ -0,0 +1,250 @@
+// Package tflitecpu is meant to implement a tfliteCPU in the ML model service
+package tflitecpu
+
+import (
+	"context"
+	fp "path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/edaniels/golog"
+	"github.com/pkg/errors"
+	"go.opencensus.io/trace"
+
+	"go.viam.com/rdk/config"
+	inf "go.viam.com/rdk/ml/inference"
+	"go.viam.com/rdk/ml/inference/tflite_metadata"
+	"go.viam.com/rdk/registry"
+	"go.viam.com/rdk/resource"
+	"go.viam.com/rdk/services/mlmodel"
+	"go.viam.com/rdk/services/vision"
+	"go.viam.com/rdk/utils"
+)
+
+var sModel = resource.NewDefaultModel("tflitecpu")
+
+func init() {
+	registry.RegisterService(vision.Subtype, resource.DefaultServiceModel, registry.Service{
```

Comment:
change this from `resource.DefaultServiceModel` to `sModel`

Reply:
Yup yup